extension/linkify-urls-in-code.js

```diff
@@ -28,7 +28,7 @@ window.linkifyURLsInCode = (() => {
 		.toArray()
 		.forEach(blob => {
 			if (hasURL(blob.innerHTML)) {
```

Comment:
Okay, however in this case I’d try to refactor the tests in a way they would actually check whether the RegExp matches actual links. In case when we simply remove `hasURL` and all of the tests related to this, the RegExp will no longer be covered by any tests.

Will push changes in a few minutes. 👨🏻‍💻